modules/server/src/network/aggregations/AggregationAccumulator.ts

```diff
@@ -13,18 +13,28 @@ type ResolveAggregationInput = {
 
 type AggregationsTuple = [Aggregations, Aggregations];
 type NumericAggregationsTuple = [NumericAggregations, NumericAggregations];
+type GenericTuple<T> = [T, T];
```

Comment:
Declaring a type alias for this, when it is only used once, is a bit overkill. Can just use `[T, T]` in place.

If this pattern is used in the future, a better name may be `TwoTuple` since tuples have arbitrary length.